notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -472,33 +472,16 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        saved_model_metadata_builder
</pre><p>this import can be removed because it's not being used anywhere in the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
It is used in **Generate model metadata for Vertex Explainable AI** section. However, I was not sure if that part is needed to be removed in this notebook as my focus was just to replace GAPICS :)